### PR TITLE
Suppress s3api calls output

### PR DESCRIPTION
**Pull request by jshoe**
- Goes to devnull like before so it doesn't clutter the logs
- Test run for this branch: https://staging.idseq.net/samples/989

### CODE REVIEW

File: idseq_dag/util/s3.py [hunk 1]
```diff
@@ -146,12 +146,15 @@ def fetch_from_s3(src,
                     os.remove(dst)
                 return None
 
+
```
Comment (tfrcarvalho) [inline idseq_dag/util/s3.py]: nit: remove blank line.

File: idseq_dag/util/s3.py [hunk 2]
```diff
@@ -146,12 +146,15 @@ def fetch_from_s3(src,
...
 def fetch_byterange(first_byte, last_byte, bucket, key, output_file):
     get_range = f"aws s3api get-object " \
                 f"--range bytes={first_byte}-{last_byte} " \
                 f"--bucket {bucket} " \
                 f"--key {key} {output_file}"
-    command.execute(get_range)
+    get_range_proc = subprocess.Popen(get_range, shell=True, stdout=subprocess.DEVNULL)
+    get_range_proc.wait()
+
```
Comment (tfrcarvalho) [inline idseq_dag/util/s3.py]: nit: remove blank line.

**Comment by jshoe** [comment]
@tfrcarvalho actually PEP 8 is two blank lines between top level functions https://www.python.org/dev/peps/pep-0008/#blank-lines Jetbrains warns on it :p

**Comment by tfrcarvalho** [comment]
Nm, then. Thought those were class methods.

**Pull request merged**